ios/FluentUI.Demo/FluentUI.Demo/Demos/PersonaBadgeCollectionDemoController.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class PersonaBadgeCollectionDemoController: UITableViewController {
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(style: .grouped)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: PersonaBadgeCollectionDemoController.largeBadgeReuseIdentifier)
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: PersonaBadgeCollectionDemoController.smallBadgeReuseIdentifier)
+
+        tableView.separatorStyle = .none
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        var cell: UITableViewCell
+        if showsFullGrid(at: indexPath) {
+            fatalError("showsFullGrid() should always return false right now!")
+        } else {
+            cell = self.tableView(tableView, badgeCellForRowAt: indexPath)
+        }
+
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch personaBadgeSize(for: section) {
+        case .small:
+            return "Small"
+        case .large:
+            return "Large"
+        }
+    }
+
+    // MARK: - Helpers
+
+    private func personaBadgeSize(for section: Int) -> MSFPersonaBadgeSize {
+        return (section % 2 == 0) ? .small : .large
+    }
+
+    private func showsFullGrid(at indexPath: IndexPath) -> Bool {
+        // TODO
```

Comment:
Actually @harrieshin pointed out that the Badge name is no good either, since that has a specific meaning for Android. So we'll need something else new still ☹️